slick/config.rb

```diff
@@ -0,0 +1,10 @@
+css_dir = "."
```

Comment:
Please remove all files from slick directory and left only slick.js and slick.min.js